react-native-libraries.json

```diff
@@ -3228,7 +3228,7 @@
     "githubUrl": "https://github.com/react-native-voice/voice",
     "ios": true,
     "android": true,
-    "expoGo": true,
+    "expoGo": false,
```

Comment:
```suggestion
```

Hello @frankcalise, nice catch, thanks for the update! 👍 

In that case we can also just remove the `expoGo` field from the package entry.